app/controllers/order_items_controller.rb

```diff
@@ -0,0 +1,42 @@
+class OrderItemsController < ApplicationController
+  before_action :set_order_item, only: %i[update destroy]
+
+  load_and_authorize_resource
+
+  def create
+    #puts "========= ADDed order_ITEM =============="
+    #current_user_order
+    @order_item = current_user_order.order_items.new(order_item_params)
```

Comment:
what if order item for book was created
will this code duplicate order items for the same book?

Reply:
If yes, then create service which will handle such case